ipfsapi/http.py

```diff
@@ -243,3 +241,7 @@ def session(self):
         yield
         self._session.close()
         self._session = None
+
+
+class HTTPClient(HTTPClientBase):
```

Comment:
That shouldn't have sunk into this commit :blush: 
I was starting to work on `asyncio` support and apparently didn't stash these lines.

Your right in that it currently doesn't make sense to do this yet, will fix.